.github/workflows/docker.yaml

```diff
@@ -0,0 +1,59 @@
+name: Build xsnippet-api Docker image
+
+on:
+  # Triggered manually
+  workflow_dispatch:
+    inputs:
+      image_name:
+        description: 'The name of the resulting Docker image'
+        required: false
+        default: 'xsnippet/xsnippet-api'
+      release_arch:
+        description: 'The CPU architecture to use when choosing the xsnippet-api binary'
+        required: false
+        default: 'x86_64'
+      release_os:
+        description: 'The OS to use when choosing the xsnippet-api binary'
+        required: false
+        default: 'linux'
+      release_tag:
+        description: 'The release tag to use when choosing the xsnippet-api binary'
+        required: false
+        default: 'latest'
+      registry:
+        description: 'The Docker registry to push the image to'
+        required: false
+        default: 'docker.io'
+      registry_username:
+        description: 'The Docker registry user name'
+        required: false
+        default: 'xsnippetci'
```

Comment:
I hope no one would parametrize the registry and and username since the password won't work with non-default values :D

Reply:
Yeah, I've only added this for testing, so that we can easily override _any_ of the available parameters!